DEVELOPERS.md

```diff
@@ -1,19 +1,26 @@
-# DEVELOPERS
+# Development information
 
-This file aims to describe an introduction for developers to work
-on the code base of radare2 project.
+This file aims to introduce developers to conventions for working on the code
+base of radare2.
+
+The GitHub issues page contains a list of all the bugs that have been reported,
+with labels to classify them by difficulty, type, milestone, etc. It is a good
+place to start if you are looking to contribute.
+
+For information about the git process, see
+[CONTRIBUTING.md](CONTRIBUTING.md#How_to_contribute).
 
 ## Documentation
 
-There is support for Doxygen document generation in this repo.
-By running `doxygen` in the root of this repository, it will autodetect the
-Doxyfile and generate HTML documentation into
-[doc/doxygen/html/index.html](./doc/doxygen/html/index.html)
+radare2 supports Doxygen document generation. By running `doxygen` in the
+repository root, it will autodetect the Doxyfile and generate HTML
+documentation at [doc/doxygen/html/index.html](doc/doxygen/html/index.html).
```

Comment:
i would discourage anyone to use doxygen, we dont use doxygen comments in r2. and the generated documentation takes GB of disk. so im fine to remove this paragraph.

also i would discourage to document function arguments, because those should be clear enough to not depend on documentation to be understood. i'm fine with one line comments clarifying something confusing or complex. but documenting all the functions is not the way to go.